src/bluetooth.rs

```diff
@@ -36,16 +36,16 @@ pub struct Device {
 impl Device {
     pub fn get_icon(name: &str) -> String {
         match name {
-            "audio-card" => String::from("󰦢 "),
+            "audio-card" => String::from("󰓃"),
             "audio-input-microphone" => String::from(""),
-            "audio-headphones" => String::from("󰥰"),
-            "battery" => String::from("󰥆"),
+            "audio-headphones" => String::from("󰋋"),
+            "battery" => String::from("󰂀"),
             "camera-photo" => String::from("󰻛"),
             "computer" => String::from(""),
             "input-keyboard" => String::from("󰌌"),
-            "input-mouse" => String::from("󰦋"),
-            "phone" => String::from("󰏳"),
-            _ => String::new(),
+            "input-mouse" => String::from("󰍽"),
+            "phone" => String::from("󰏲"),
+            _ => String::from("?"),
```

Comment:
Would be nice to have empty string here or something else instead of `?`, what do you think ?